create-container.sh

```diff
@@ -5,15 +5,81 @@ set -e
 # Uncomment the following line to debug the script
 # set -x
 
+function print_help {
+    echo """
+    Usage: devenv [subcommand]
+
+A bash script to create and manage development environments using privileged LXC linux containers.
+
+Subcommands:
+    init - Generate the default configuration file. Create the .devenv
+
+
+More info: https://github.com/coopdevs/devenv
+    """
+}
+
+function init {
+    CONFIG="$1"
+    if [ -f "$CONFIG" ]; then
+      printf "The devenv config file already exists. Execute:\n\n$ cat .devenv\n"
+      exit 1
+    else
+      cat > "${CONFIG}" << EOF
+# File created with devenv init command
+
+NAME="<container name>"
+DISTRIBUTION="<SO distribution>"
+RELEASE="<SO release>"
+ARCH="<SO arch>"
+HOST="local.$NAME.coop"
+
+# Optional -- To create a new user and group
+DEVENV_USER="<user that will own the project>"
+DEVENV_GROUP="<group that will own the project>"
+
+# Optional -- To mount a project.
+# Make sure that the directory "../$PROJECT_NAME" exists
```

Comment:
If it's optional, we can't ask to make sure that it exists. 
```suggestion
# If you don't need a shared dir between host and guest, just
# comment the lines or unset them. Empty string doesn't work.
# Otherwise, if you need the shared mount, set the vars below and
# make sure that the directory "../$PROJECT_NAME" exists
```